### ISSUE TITLE
Replace ogr2ogr with GDAL (was: pyshp)

### ISSUE THREAD

**Issue by gaurav** [MapofLife/MOL#2]
pyshp is a Python-only library for manipulating shapefiles. Since we're already incorporating it to create a dummy DBF file for @eightysteele's shapefile-to-PostGIS hack, it might make sense to read the fields directly from the shapefiles using pyshp instead of relying on GDAL being installed to handle the DBF-to-CSV transformation. It would also be handy in evolving loader.py into an independent  program which we could distribute to data providers, who could use it to validate their files before uploading them to MoL.

Obviously, a _very_ low priority task for the foreseeable future. I just wanted to write it down so I wouldn't forget :).

**Comment by gaurav** [comment]
We've temporarily given up on pyshp, and are back to using ogr2ogr for all tasks. I would like to see EITHER:
1. We replace ogr2ogr with [GDAL](http://pypi.python.org/pypi/GDAL/), so we don't have to figure out paths and depend on ogr2ogr's command line interface not changing, OR:
2. We replace ogr2ogr with pyshp after all, since it is a pure-Python library and so ought to be incredibly portable.

Either way, everything's working well in this department for now, so definitely very low priority.

**Comment by eightysteele** [comment]
No longer relevant with new loader architecture. Closing issue.
